pkg/resource/training_job/testdata/test_suite.yaml

```diff
@@ -210,4 +249,196 @@ tests:
             - operation: StopTrainingJobWithContext
         invoke: Delete
         expect:
-          error: nil
\ No newline at end of file
+          error: nil
+  - name: "Training job Update tests"
+    description: "Tests for update operation"
+    scenarios:
+      - name: "Update=RemoveProfilerBothParams"
+        desciption: "This test checks if the Controller can remove the profiler properly"
+        given:
+          desired_state: "v1alpha1/update/desired/removeProfilerboth.yaml"
+          latest_state: "v1alpha1/update/latest/profilerHappyBase.yaml"
+          svc_api: 
+            - operation: UpdateTrainingJobWithContext
+              output_fixture: "sdkapi/update/update_sucess.json"
+            - operation: DescribeTrainingJobWithContext
+              output_fixture: "sdkapi/update/describe/profiler_removal.json"
+        invoke: Update
+        expect:
+          latest_state: "v1alpha1/update/observed/removeProfilerBoth.yaml"
+          error: training job is updating
+      - name: "Update=RemoveProfilerRule"
+        desciption: "This test checks if the Controller can remove the profiler properly"
+        given:
+          desired_state: "v1alpha1/update/desired/removeProfilerRule.yaml"
+          latest_state: "v1alpha1/update/latest/profilerHappyBase.yaml"
+          svc_api: 
+            - operation: UpdateTrainingJobWithContext
+              output_fixture: "sdkapi/update/update_sucess.json"
+            - operation: DescribeTrainingJobWithContext
+              output_fixture: "sdkapi/update/describe/profiler_removal.json"
+        invoke: Update
+        expect:
+          latest_state: "v1alpha1/update/observed/removeProfilerRule.yaml"
+          error: training job is updating
+      - name: "Update=RemoveProfilerConfig"
+        desciption: "This test checks if the Controller can remove the profiler properly"
```

Comment:
nit: looks like description is spelled incorrectly here for all description fields, if oyu have time do you mind making the descriptions have more detail for the specific test. We can infer the reason from the test-name but a lot ofthese have the same description

Reply:
Yea I will update them for completeness